src/MovingHarmonicPotential.cc

```diff
@@ -3,50 +3,45 @@
 // Part of azplugins, released under the BSD 3-Clause License.
 
 /*!
- * \file ImplicitEvaporator.cc
- * \brief Definition of ImplicitEvaporator
+ * \file MovingHarmonicPotential.cc
+ * \brief Definition of MovingHarmonicPotential
  */
 
-#include "ImplicitEvaporator.h"
+#include "MovingHarmonicPotential.h"
+
+namespace hoomd
+    {
 
 namespace azplugins
     {
 /*!
  * \param sysdef System definition
  * \param interf Position of the interface
  */
-ImplicitEvaporator::ImplicitEvaporator(std::shared_ptr<SystemDefinition> sysdef,
-                                       std::shared_ptr<Variant> interf)
+MovingHarmonicPotential::MovingHarmonicPotential(std::shared_ptr<SystemDefinition> sysdef,
+                                                 std::shared_ptr<Variant> interf)
     : ForceCompute(sysdef), m_interf(interf), m_has_warned(false)
     {
     // allocate memory per type for parameters
     GPUArray<Scalar4> params(m_pdata->getNTypes(), m_exec_conf);
     m_params.swap(params);
-
-    // connect to type change to resize type data arrays
-    m_pdata->getNumTypesChangeSignal()
-        .connect<ImplicitEvaporator, &ImplicitEvaporator::reallocateParams>(this);
     }
 
-ImplicitEvaporator::~ImplicitEvaporator()
-    {
-    m_pdata->getNumTypesChangeSignal()
-        .disconnect<ImplicitEvaporator, &ImplicitEvaporator::reallocateParams>(this);
-    }
+MovingHarmonicPotential::~MovingHarmonicPotential() { };
```

Comment:
`;` not needed after function definition.
```suggestion
MovingHarmonicPotential::~MovingHarmonicPotential() { }
```